### DISCUSSION TITLE
How to show `References Buffer`

### DISCUSSION THREAD

**Discussion by azzamsa** [NeogitOrg/neogit#1074] [Q&A] [answered]
Hi, 👋.

Coming from Magit, How to show [References Buffer](https://magit.vc/manual/magit/References-Buffer.html).
Usually, in my workflow, I press `y` in the Log buffer to call `magit-show-refs`.
Why? I wanted to know the branches and remotes in my Upstream, Origin, etc.

![image](https://github.com/NeogitOrg/neogit/assets/17734314/55a92e2d-a33f-4dfa-a8a1-aa3b5477b753)

**Comment by CKolkey** [comment, marked as answer]
Hi! Its available in the `nightly` branch, and requires nvim nightly. It uses some the folding api's that dont exist in nvim 0.9. 

I haven't hooked it up to the log buffer yet, but that should be simple. However, it is available from the status buffer :)

**Reply by azzamsa** [reply]
I think I will wait until Nvim v10 released :)
